programmers-coding-test/kimjiwon/알고리즘/거스름돈.js

```diff
@@ -0,0 +1,32 @@
+function giveChange(customerMoney, goodsPrice) {
+    const unitMoney = [10000 ,5000 ,1000, 500, 100, 50, 10];
+    const changeUnitMoney = new Map();
+
+    let sum = 0,
+    change = customerMoney - goodsPrice;
+
+    unitMoney.forEach(el => {
+        if (calcurate(change, el)) {
+            changeUnitMoney.set(`${el}`, calcurate(change, el));
+            change -= (el * calcurate(change, el)); 
+        }
+    });
```

Comment:
el 보다 명확히 지칭하는 변수명을 써주었으면 한다. Element는 최대한 지양하자.
unitMoney => unitMoneys로 복수형태로 만들고 반복문 안에서 element를 단수형으로 불러왔다.

```suggestion
    unitMoneys.forEach(unitMoney => {
        if (calcurate(change, unitMoney)) {
            changeUnitMoney.set(`${unitMoney}`, calcurate(change, unitMoney));
            change -= (unitMoney * calcurate(change, unitMoney)); 
        }
    });
```